### DISCUSSION TITLE
Episode numbering when a season has 3 or 4 digit number of episodes

### DISCUSSION THREAD

**Discussion by Fidelisraphael** [Flexget/Flexget#2847] [Q&A]
So I'm using flexget and tvdb_ep_id and move command to change the file name. But tvdb_ep_id returns episode 100 as 00, 101 as 01, etc. How do I change it so that it can display a 3 (or even a 4) digit episode numbering? It also sometimes fails the lookup for tvdb_ep_name and tvdb_absolute_number even though thetvdb has this information available. Is that due to too many requests in a short amount of time, and anything that I can change in order to get that information as well except to run it several times?

This is in regards to anime, with specifically the show Boruto: Naruto Next Generations giving me issues currently.

There seems to be a similar thread on the old forums, are they coming back or are they lost forever?
https://discuss.flexget.com/t/matching-series-season-and-episode-number/1145/1

I've even tried to set up an if-loop in order to get the information elsewhere if tvdb fails like so, but to no avail.

{% if tvdb_ep_id|default(False) %}{{ tvdb_ep_id }}{% elif trakt_ep_id|default(False) %}{{ trakt_ep_id }}{% elif series_id|default(False) %}{{ series_id }}{% elif tvdb_season|default(False) and tvdb_episode|default(False) %}S{{ tvdb_season }}E{{ tvdb_episode }}{% elif trakt_season|default(False) and trakt_episode|default(False) %}S{{ trakt_season}}E{{ trakt_episode }}{% elif series_season|default(False) and series_episode|default(False) %}S{{ series_season }}E{{ series_episode }}{% elif series_episode|default(False) %}S00E{{ series_episode }}{% else %}SXXEYY{% endif %}

**Comment by ksurl** [comment]
You can use {{ series_id|pad(3) }} if it's actually getting the correct episode number
